src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -62,8 +62,10 @@ namespace wallet_rpc
     struct request
     {
       uint32_t account_index;
+      std::set<uint32_t> address_index;
```

Comment:
Maybe best to call it address_indices ? Also there's a RPC version number for the wallet now too. Seemed like a good idea at the time :)